Modules/Loadable/Markups/qSlicerMarkupsReader.cxx

```diff
@@ -76,21 +76,25 @@ vtkSlicerMarkupsLogic* qSlicerMarkupsReader::markupsLogic()const
 //-----------------------------------------------------------------------------
 QString qSlicerMarkupsReader::description()const
 {
-  return "MarkupsFiducials";
+  return "Markups";
 }
 
 //-----------------------------------------------------------------------------
 qSlicerIO::IOFileType qSlicerMarkupsReader::fileType()const
 {
-  return QString("MarkupsFiducials");
+  return QString("Markups");
 }
 
 //-----------------------------------------------------------------------------
 QStringList qSlicerMarkupsReader::extensions()const
 {
-  return QStringList()
-    << "Markups Fiducials (*.fcsv)"
-    << " Annotation Fiducial (*.acsv)";
+  return QStringList() << " Markups (*.json)";
```

Comment:
Related to the comment I made below, the use of JSON-Schema can make it truly unambiguous, since you could reference the (versioned) schema URL (in the Slicer github repo) directly from JSON markup document, and there are many tools that can validate a document against the schema. This won't help with the reader, unless it can examine the content of the file, but will make those documents more portable outside of Slicer.

About the custom extensions workaround - I am curious - are those that are currently used documented in a single place somewhere in Slicer documentation?

Reply:
About the page, I think we should have something like that to establish the conventions, document the file organization, and manage conflict resolution for composite extension assignment. Otherwise, adding new "flavors" of formats and composite extensions might be hard for developers adding new ones, for those who create those documents from outside of Slicer to be consumed in Slicer, or for those who want to interpret Slicer-generated documents outside of Slicer.

Reply:
For encouraging standard behavior it could be useful.
Please note that these secondary extension parts are only for additional information and have no other consequences. The file will still be a json/nrrd/etc, it just allows Slicer to make loading them easier for the user. So files created externally with only json/nrrd can be loaded just as well, but with an extra option selection in the load window, as it would be required otherwise. 3rd party developers don't need to be aware of any of this.

Reply:
Yes, I think I understand, but I am also thinking about the situation where someone wants to, for example, convert a time-resolved image created some non-Slicer software into a format that would be understood by the Sequences module. Or consume Slicer-specific segmentation file in a custom software that does not use Slicer IO. It would help to have a central place (linked from https://www.slicer.org/wiki/Documentation/4.8/SlicerApplication/SupportedDataFormat ?) where documentation on the details of those custom formats.

Reply:
I still agree, such a resource would be useful. The reason I'm reluctant to create simple wiki pages is that most (!) of our wiki pages are already outdated, and it is a huge task to keep them up-to-date, with which we apparently cannot keep up. So I think any new documentation should be dynamic if possible. There are efforts in this direction. Unfortunately a bit stalled, but that seems to be the only sustainable way.

Reply:
I agree that the [Supported Data Format page](https://www.slicer.org/wiki/Documentation/Nightly/SlicerApplication/SupportedDataFormat) should be improved. We could describe if a particular file format can be interpreted in different ways (e.g., nrrd file can be loaded as a scalar volume, labelmap volume, diffusion volume, segmentation, multivolume, volume sequence, or grid transform). We could also describe all the known DICOM plugins.